CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ endif()
 # Let windows builds auto-export dll symbols
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
This should probably not be part of the PR :)

Reply:
Good point.